src/user_crate/target.rs

```diff
@@ -0,0 +1,80 @@
+/*! PL/Rust adopts the tactic of always explicitly specifying which target to build for.
+    This prevents using the "fallback" logic of Cargo leaving builds in an unlabeled directory.
+    This is a precaution as PL/Rust is a cross-compiler.
+    so a normal build-and-test cycle may create artifacts for multiple targets.
+!*/
```

Comment:
I think the `!` is unnecessary but I ain't mad if it stays!

Reply:
It feels nice and symmetric? :shrug: